api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -750,22 +750,52 @@ message ScopedRds {
       [(validate.rules).message = {required: true}];
 }
 
+// [#next-free-field: 6]
 message HttpFilter {
   option (udpa.annotations.versioning).previous_message_type =
       "envoy.config.filter.network.http_connection_manager.v2.HttpFilter";
 
+  // [#not-implemented-hide:] Configuration source specifier for the late-bound
+  // filter configuration. The HTTP Listener is warmed until all the initial
+  // filter configurations are received, unless the flag to apply the default
+  // configuration is set. Subsequent filter updates are atomic, and apply to
```

Comment:
Nit: atomic on a per-worker basis. It's still plausible to have a series of ordered requests at an Envoy experience [old,new,old,new] config.

Reply:
Good point. This seems to be a general pattern for all xDS but good to point out.